src/payment/strategies/paypalCommerce/paypalCommerce-script-loader.ts

```diff
@@ -0,0 +1,27 @@
+import { ScriptLoader } from '@bigcommerce/script-loader';
+
+import { PaymentMethodClientUnavailableError } from '../../errors';
+
+import { PaypalCommerceHostWindow, PaypalCommerceSDK } from './paypalCommerce-sdk';
+
+export default class PaypalCommerceScriptLoader {
```

Comment:
Can we have kebab case for file and folder names? 💅

Reply:
Sure!